src/file-manager/fm-desktop-icon-view.c

```diff
@@ -882,14 +882,14 @@ fm_desktop_icon_view_supports_uri (const char *uri,
 
 static CajaViewInfo fm_desktop_icon_view =
 {
-    FM_DESKTOP_ICON_VIEW_ID,
-    "Desktop View",
-    "_Desktop",
-    N_("The desktop view encountered an error."),
-    N_("The desktop view encountered an error while starting up."),
-    "Display this location with the desktop view.",
-    fm_desktop_icon_view_create,
-    fm_desktop_icon_view_supports_uri
+    .id = FM_DESKTOP_ICON_VIEW_ID,
+    .view_combo_label = "Desktop View",
+    .view_menu_label_with_mnemonic = "_Desktop",
```

Comment:
To enable text translation, text should be marked with N_(TEXT).

Reply:
![Captura de Pantalla 2019-08-21 a les 16 58 50](https://user-images.githubusercontent.com/10171411/63443496-2e438b00-c435-11e9-901a-1ebf8f581ac0.png)